README.md

```diff
@@ -1,3 +1,47 @@
-[![Build status](https://ci.appveyor.com/api/projects/status/thjw94949tm5lbw5/branch/master?svg=true)](https://ci.appveyor.com/project/nvborisenko/client-net/branch/master)
-[![NuGet Badge](https://buildstats.info/nuget/reportportal.client)](https://www.nuget.org/packages/reportportal.client)
-[![Coverage](https://codecov.io/gh/reportportal/client-net/branch/master/graph/badge.svg)](https://codecov.io/gh/reportportal/client-net)
+[![Build status](https://ci.appveyor.com/api/projects/status/thjw94949tm5lbw5/branch/master?svg=true)](https://ci.appveyor.com/project/nvborisenko/client-net/branch/master) [![NuGet Badge](https://buildstats.info/nuget/reportportal.client)](https://www.nuget.org/packages/reportportal.client) [![Coverage](https://codecov.io/gh/reportportal/client-net/branch/master/graph/badge.svg)](https://codecov.io/gh/reportportal/client-net)
+
+# Project Name
+API client for https://reportportal.io
```

Comment:
I suggest to organize it as a link to report portal.

Reply:
@Aleh-Yanushkevich  Could you please add a bit more details how it should look like? Thank you :)